src/web/tools/KeyboardEventManager.ts

```diff
@@ -0,0 +1,93 @@
+import { AdaptedEvent, EventTypes } from '../interfaces';
+import EventManager from './EventManager';
+import { PointerType } from '../../PointerType';
+
+export default class KeyboardEventManager extends EventManager<HTMLElement> {
+  private activationKeys = ['Enter', ' '];
+  private cancelationKeys = ['Tab'];
+  private isPressed = false;
+
+  private keyDownCallback = (event: KeyboardEvent): void => {
+    if (this.cancelationKeys.indexOf(event.key) !== -1 && this.isPressed) {
+      this.dispatchEvent(event, EventTypes.CANCEL);
+      return;
+    }
+
+    if (this.activationKeys.indexOf(event.key) === -1) {
+      return;
+    }
+
+    this.dispatchEvent(event, EventTypes.DOWN);
```

Comment:
As an alternative we could do:

```suggestion
    if (this.cancelationKeys.indexOf(event.key) !== -1 && this.isPressed) {
      this.dispatchEvent(event, EventTypes.CANCEL);
    } else if (this.activationKeys.indexOf(event.key) !== -1) {
      this.dispatchEvent(event, EventTypes.DOWN);
    }
```
which is a bit shorter. It's up to you though 😅